src/ground-commander/ground-commander-main.cpp

```diff
@@ -36,24 +36,21 @@ int main()
 #endif
 
     Shakespeare::log(Shakespeare::NOTICE, GC_LOGNAME, "Waiting for commands to send or satellite data");
-     	
     while (true)
     {
         int result_bytes = read_results();
-	
         if (result_bytes > 0) {
             //get result buffers
             memset(gc_log_buffer,0,CS1_MAX_LOG_ENTRY);
             snprintf(gc_log_buffer,CS1_MAX_LOG_ENTRY,"Got bytes from Ground Netman: %d", result_bytes);
             Shakespeare::log(Shakespeare::NOTICE, GC_LOGNAME, gc_log_buffer );
-
-            perform(result_bytes);
+	    Process_results(result_bytes);
```

Comment:
Small style comment, should be process_results for consistency with the other function names